### ISSUE TITLE
SPRINT 43 PRIORITIES

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#21358]
## Instructions for first few sprints using this priority document:

- This is a living document that everyone is empowered to keep updated.  
- If you complete a task, please come here to check if off.  If you add a new issue or run across something that needs to be discussed in the next grooming session, please include add it to Discussion Topics/Grooming.

## Sprint 43 Priorities

1.  Yellow Ribbon: Implement Google Analytics data layer event code for Yellow Ribbon (remaining from Sprint 41 & 42)
2.  Find a VA Form: Test different search variations against changes Zach made in Sprint 42, make adjustments
3.  Find a VA Form: Monitor bugs CMS, Lighthouse, VA Forms Mgrs are working on (including Zach consuming the new field Lighthouse added for Admin so that the "related to" information is always populated)
4. Yellow Ribbon: Verify events, request Domo Dashboard (blocked by 1)
5. Update CMS with "When to Use" and Online Tool description data where needed, create/modify Best Bets where needed.
6. Find a VA Form: Design consistency for VA.gov searches
7. 508 tech debt 🔴 Marci - comment on two In Progress 508's for Trevor & Josh to review.

<hr>

#### Zach (@zacharymorel )
- [x] Test search variations against changes made in Sprint 42
- [x]  [BUG] Find a VA Form: "Related to" information missing on some results #19561 - make FE changes to consume new field Mark Viterna added 

#### Cassandra (@callen2563)
- Search UI Consistency #21055 

#### John (@johnhashva 
- [x] Create/modify Best Bets for Search.gov where needed.

#### Marci
- [x] Update CMS with "When to Use" info, create requests for Content team to review drafts
- [x] Review Prod CMS for any new forms needing details pages
- [x] Organize product folders in GitHub
- [x] Monitor bugs Lighthouse is working on
- [x] Monitor Analytics KPI dashboard request for Decision Tools Team - Find a VA Form #19689 

<hr>

#### Zach
- [x] Analytics Review on Staging - Jon W added more suggestions 3/8/2021  #18234  
- [x] Re-deploy search tips to Prod post-Analytics review  #18234  

#### John
- [ ] N/A

#### Marci
- [ ] Verify new events, create Domo dashboard/reports request for Yellow Ribbon once event capture tags are in Prod

### 🛠️ Technical Debt / A11y Issues
**Our goal is to include 1-2 tech debt issues into each sprint.**

#### Zach
- [x] sitewide 508-defect-2 [COGNITION, SCREENREADER]: Pagination SHOULD use meaningful landmarks and aria techniques #18398 - Note: Josh K back from PTO
- [ ] sitewide 508-defect-2 [COGNITION, SCREENREADER]: Heading SHOULD read as a single item #18389
  - Note: Josh K back from PTO
-  [x] 508-defect-2 [SCREENREADER]: Headings SHOULD be used semantically #18429
-  [ ]  508-defect-3 [COGNITION, SCREENREADER]: Lists SHOULD be announced as lists #18415
  🔴 Marci to schedule recurring time with Josh Kim on Mondays.
   
<hr>

#### CMS
None at this time

#### Lighthouse 
  
- [ ] [Bug] Find a VA form search is showing duplicate listing of VA Form 10182 #17484 
    - 2021/03/11 - Mark V will promote new ID field next Thu
    - 2021/03/05 - Mark V and Zach are collaborating on a solution
    - 2021/02/26 - Asked for this issue to be lower priority than #19561
    - 2021/02/26 - Met with Lighthouse, Steve Wirt, Zach to discuss.
 
- [x] [BUG] Find a VA Form: "Related to" information missing on some results #19561
    - 2021/03/11 - Mark V will promote new Administration field next Tue
   - 2021/03/04 - Mark V and Zach are collaborating on a solution.  Mark has made his changes & Zach will need to make changes to his code to consume the new data.
   - 2021/02/26 - Asked for this issue to be higher priority than #17484
   - 2021/02/24 Per Nichole, prioritized for sprint that is now starting.

#### VA Forms Managers

- [ ] Find a VA Form [BUG] - Invalid character in VA DB PDF URL is causing "not found" error. #18926

<hr>
